sdk/samples/iot/README.md

```diff
@@ -326,6 +327,15 @@ This section provides an overview of the different samples available to run and
 
   </details>
 
+### IoT Plug and Play with Provisioning Sample
+
+- *Executable:* `paho_iot_pnp_with_provisioning_sample`
+
+  This sample has the same functionality as the above `paho_iot_pnp_sample` but uses the Device Provisioning Service for authentication. The same steps above may be followed for interacting with the sample, but you must instead set the following environment variables for authentication:
+  - `AZ_IOT_PROVISIONING_ID_SCOPE`
+  - `AZ_IOT_PROVISIONING_REGISTRATION_ID`
+  - `AZ_IOT_DEVICE_X509_CERT_PEM_FILE_PATH`
+
```

Comment:
I'm not sure if I would include these here since none of the sample descriptions includes them.  All the environment variables are specified further below.  

What about this, which incudes links to the code like the other sample descriptions and a link to the above referenced Plug and Play Sample.  It follows a similar structure to the other descriptions:

This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_provisioning_pnp_sample.c) has the same functionality as the [IoT Hub Plug and Play Sample](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/samples/iot#iot-hub-plug-and-play-sample) but uses the Azure Device Provisioning Service. X509 authentication is used.

Reply:
I believe the link verification will break if I do that right now. I'll have to add the links in another PR but good suggestion.